core/arch/instr_shared.c

```diff
@@ -2159,6 +2159,37 @@ instr_set_our_mangling(instr_t *instr, bool ours)
         instr->flags &= ~INSTR_OUR_MANGLING;
 }
 
+bool
+instr_is_our_mangling_epilogue(instr_t *instr)
+{
+    ASSERT(!TEST(INSTR_OUR_MANGLING_EPILOGUE, instr->flags) ||
+           instr_is_our_mangling(instr));
```

Comment:
This should be in the header comment: as written a caller would expect this to return false for !instr_is_our_mangling with INSTR_OUR_MANGLINE_EPILOGUE set, instead of an assert firing.  The assumption/requirements should be documented: this routine requires the caller to already know that instr is our-mangling.

Reply:
Done